File: 3455-minimum-length-of-string-after-operations/minimum-length-of-string-after-operations.java
class Solution {
    public int minimumLength(String s) {
        int[] st = new int[26];
        int k = 0;
        for (char c: s.toCharArray()) st[c-'a']++;
        for (int e: st) k+= (e != 0 ? ((e & 1) == 1 ? 1 : 2) : 0);
        return k;
    }
}

File: 3455-minimum-length-of-string-after-operations/minimum-length-of-string-after-operations.cpp
class Solution {
public:
    int minimumLength(string s) {
        vector<int> st(26);
        for (char& c: s)    st[c-'a']++;
        int k = 0;
        for (int& e: st)   e ? ((e & 1) ? k++ : k += 2) : 0;
        return k;
    }
};